packages/foreman/rubygem-actioncable/rubygem-actioncable.spec

```diff
@@ -3,33 +3,32 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name actioncable
+%global gem_require_name %{gem_name}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 6.0.2.2
-Release: 2%{?dist}
+Version: 6.0.3.1
+Release: 1%{?dist}
 Summary: WebSocket framework for Rails
 Group: Development/Languages
 License: MIT
-URL: http://rubyonrails.org
+URL: https://rubyonrails.org
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 2.5.0
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(actionpack) = 6.0.2.2
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+Requires: %{?scl_prefix}rubygem(actionpack) = 6.0.3.1
 Requires: %{?scl_prefix}rubygem(nio4r) >= 2.0
 Requires: %{?scl_prefix}rubygem(nio4r) < 3
 Requires: %{?scl_prefix}rubygem(websocket-driver) >= 0.6.1
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}ruby >= 2.5.0
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel 
```

Comment:
any reason there is a trailing space now?

Reply:
Gem2rpm said so? Will remove